recipes-kernel/linux/linux-renesas_%.bbappend

```diff
@@ -13,9 +12,11 @@ SRC_URI += "file://WIRELESS.cfg"
 
 FILESEXTRAPATHS_prepend := "${THISDIR}/dts:"
 SRC_URI += "file://0001-remove-sdhi1-uhs.patch"
+SRC_URI += "file://0002-add-vsc8531-userspace-dts.patch"
+SRC_URI += "file://0003-add-mistysom-phy-mode-change.patch"
+SRC_URI += "file://0004-remove-phy-interrupts-dts.patch"
```

Comment:
Is it possible that you would combine all these patches into a single patch?

Reply:
`0002` overrides the actual phy node,
`0003` edits the dtsi to set the phy-mode to `rgmii-id`
`0004` edits a different dtsi to remove the interrupt settings.
Would you suggest to combine them into a single file even though they di different things (to different files, as-is)?

Reply:
I am also patching different files in `0001-remove-sdhi1-uhs.patch`, but I they were for the same purpose. I thought maybe it makes sense to bind patches together. As if it doesn't make sense to change one if the other is not also changing.

Reply:
>As if it doesn't make sense to change one if the other is not also changing.

If you look at it this way, fair enough, point taken! I've merged the patches into a single file! Looks much cleaner :+1:

Reply:
> @mistyron Let's merge develop into phy-enable-working2 so we could see a yocto build test.

Ok, yeah I see you merrged the other PR, I'll pull, merge & push